### DISCUSSION TITLE
how to disable an input based on dynamic items count

### DISCUSSION THREAD

**Discussion by hibasalem** [vueform/vueform#335] [Questions] [answered]
in the following example, i get the items of selectB through an API based on selectA value , and I need to disable selectB  when there is no options returned from the API 
however, I couldn't find a callback or event to call once the items call is done , I did try the `updated` event and the `beforeUpdate` but those doesn't get triggered 
```
'schema': {
  'selectA': {
    'type': 'select',
    'closeOnSelect': true,
    'items': [
      // items list 
    ],
    'search': true,
    'native': false,
    'inputType': 'search',
    'autocomplete': 'off',
    'floating': false,
    'appendToBody': true,
  },
  'selectB': {
    'type': 'select',
    'closeOnSelect': true,
    'items': 'api?selectA={selectA}',
    'search': true,
    'native': false,
    'inputType': 'search',
    'autocomplete': 'off',
    'floating': false,
    'appendToBody': true,
    // disable this ????? 
  },
}
```

**Comment by adamberecz** [comment, marked as answer]
The[ `disabled`](https://vueform.com/reference/select-element#option-disabled) accepts an array of [conditions](https://vueform.com/docs/conditional-rendering#element-conditions), which can include a [function](https://vueform.com/docs/conditional-rendering#functional-conditions). You can use that to dynamically determine whether `selectB` should be disabled. Here's an example: https://stackblitz.com/edit/github-yevtas?file=src%2FApp.vue

**Reply by hibasalem** [reply]
this worked thanks
